src/formats/solidDatasetAsTurtle.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2022 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import type { SolidDataset } from "../interfaces";
+import { prefixes as defaultPrefixes } from "./prefixes";
+import { NamedNode, Writer } from "n3";
+import { toRdfJsDataset } from "../rdfjs";
+
+/**
+ * A function to serialise a Solid Dataset as Turtle
+ * 
+ * @param dataset The Dataset to serialize as Turtle
+ * @param options.prefixes The Prefixes to use for Turtle serialisation (defaulting to a set of well known prefixes)
+ * @param options.thing Restricts serialisation to the part of a dataset related to the thing
```

Comment:
This is going to look like two different params in the docs. Maybe use a bullet list but a single `@param` tag ?

Reply:
I still can't really find the relevant bits in [TypeDoc's documentation](https://typedoc.org/guides/doccomments/#%40param-%3Cparam-name%3E) (how ironic ^^,)... but it looks like this pattern is what's expected if you want to explain options parameters without creating an interface: https://github.com/TypeStrong/typedoc/issues/497#issuecomment-315555494